librz/reg/profile.c

```diff
@@ -397,7 +397,6 @@ RZ_API char *rz_reg_parse_gdb_profile(const char *profile_file) {
 				str = rz_file_slurp(file, NULL);
 				free(file);
 			}
-			free(base);
```

Comment:
?? why?

Reply:
![image](https://user-images.githubusercontent.com/203261/114672970-4e297000-9d38-11eb-8544-ea0ace2c5b05.png)

Reply:
Though you are probably right and it's false positive in this concrete case, I will remove this commit.